modules/hal_nxp/mcux/CMakeLists.txt

```diff
@@ -16,7 +16,6 @@ if(CONFIG_SOC_LPC11U66
     OR CONFIG_SOC_MIMX8UD7_ADSP
     OR CONFIG_SOC_MIMX9352_A55
     OR CONFIG_SOC_MIMX9596_A55
-    OR CONFIG_SOC_MIMX9596_M7
```

Comment:
AFAIK this'll break the other EDMA driver which is why it was introduced via https://github.com/zephyrproject-rtos/zephyr/pull/87817. Is this really necessary?

Reply:
Indeed, the fsl_edma4 driver is used to support edma3, edma4 and edma5. I've been watching Zephyr edma driver for a while, and I would expect the same in Zephyr. @yongxu-wang15 could you extend the driver to support edma4 as well?

Reply:
Hi @LaurentiuM1234 , the fsl_edma_v2, and fsl_edma_soc_v2 can be added in sdk-ng, it will simplify the cases such as IMX95 M7 support. The only change is the contribution workflow, the patch need to be pushed to MCUX SDK repo, then synced to hal_nxp. thanks.